AutomatticTracks/src/main/java/com/automattic/android/tracks/crashlogging/internal/SentryCrashLogging.kt

```diff
@@ -55,8 +56,11 @@ internal class SentryCrashLogging constructor(
         )
     }
 
-    private fun mergeKnownKeysWithValues(event: SentryEvent) = dataProvider.extraKnownKeys()
-        .associateWith { knownKey -> event.getExtra(knownKey).toString() }
```

Comment:
This is core problem with previous implementation. When `SentryEvent` didn't have applied value for a `knownKey`, we received `null` which we later mapped to string (like `null.toString()`) so we've ended up with `"null"` instead of `null`